src/MessagingAPI.ts

```diff
@@ -24,6 +24,7 @@ export interface MessagingAPI {
     /** Get total number of unseen messages from all conversations the user has joined */
     getTotalUnseenMessages(): number
 
+    getProfileInfo(userId: string): Promise<{ displayName?: string; avatarUrl?: string }>
```

Comment:
Might be good to have an alias for this type.

Reply:
I think this one is missing:
```suggestion
    getProfileInfo(userId: string): Promise<ProfileInfo>
```